datanode/entities/market.go

```diff
@@ -43,6 +43,7 @@ type Market struct {
 	State                         MarketState
 	MarketTimestamps              MarketTimestamps
 	PositionDecimalPlaces         int
+	LpPriceRange                  float64
```

Comment:
Also I don't see any changes to the database schema, you'll need an additional column to actually store this in (see sqlstore/migrations)

Reply:
yes exactly :) though as mentioned above I think we'd be better off with a NUMERIC type with has arbitrary precision e.g. 

 NUMERIC(32, 16)

Which is a bit overkill gives you 16 digits either side of the decimal place :)